packages/runtime/test/transport/messages.test.ts

```diff
@@ -272,6 +275,73 @@ describe("Message errors", () => {
         expect(result).toBeAnError("The recipient does not match the Request's peer.", "error.runtime.validation.invalidPropertyValue");
     });
 
+    test("should throw correct error message for trying to send a Message with a Request that expired", async () => {
```

Comment:
Please adjust the test name to `"should throw correct error for trying to send a Message with an expired Request"`.

("should throw correct error" instead of "should throw correct error message" for consistency with test names above and "an expired Request" instead of "a Request that expired" as it is grammatically preferable.)